app/src/main/java/com/example/chienmusic/data/model/Track.java

```diff
@@ -36,6 +40,8 @@ public Track(Builder builder) {
         private int mDuration;
         private int mDownloadCount;
         private boolean mIsDownloadable;
+        private User mUser;
+        private String mArtist;
 
         public Builder withArtworkUrl(String artworkUrl) {
```

Comment:
Co the bo Builder di em.

Reply:
nhưng nhiều tham số quá cũng bỏ được à anh